content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -377,14 +377,12 @@ Here are some examples of device plugin implementations:
 * The [AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
 * The [Intel device plugins](https://github.com/intel/intel-device-plugins-for-kubernetes) for
   Intel GPU, FPGA, QAT, VPU, SGX, DSA, DLB and IAA devices
-* The [KubeVirt device plugins](https://github.com/kubevirt/kubernetes-device-plugins) for
-  hardware-assisted virtualization
 * The [NVIDIA GPU device plugin for Container-Optimized OS](https://github.com/GoogleCloudPlatform/container-engine-accelerators/tree/master/cmd/nvidia_gpu)
-* The [RDMA device plugin](https://github.com/hustcat/k8s-rdma-device-plugin)
 * The [SocketCAN device plugin](https://github.com/collabora/k8s-socketcan)
-* The [Solarflare device plugin](https://github.com/vikaschoudhary16/sfc-device-plugin)
-* The [SR-IOV Network device plugin](https://github.com/intel/sriov-network-device-plugin)
+* The [SR-IOV Network device plugin](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin) for
```

Comment:
I suggest we add a disclaimer that this link is not to official Kubernetes project code, despite the use of Kubernetes in then name of the GitHub organization.

Reply:
you mean disclaimer  that k8snetworkplumbingwg community is not  hosting official Kubernetes project code?

Reply:
The disclaimer should make it clear that k8snetworkplumbingwg is not a Kubernetes project GitHub organization. and that despite use of the K8s trade mark, the code _is_ third party.

Reply:
As an alternative, we could omit the link to the SR-IOV device plugin pending a resolution on the trademark issue.